pkg/utils/utils.go

```diff
@@ -574,8 +574,10 @@ func setVfsAdminMac(iface *sriovnetworkv1.InterfaceExt) error {
 		if err := netlink.LinkSetVfHardwareAddr(pfLink, vfID, vfLink.Attrs().HardwareAddr); err != nil {
 			return err
 		}
-		if err = Unbind(addr); err != nil {
-			return err
+		if vfLink.Attrs().EncapType == "infiniband" {
```

Comment:
this method is only called for ethernet link type, please see https://github.com/openshift/sriov-network-operator/blob/b2b495af316f49434b9ae841b701923cf629246d/pkg/utils/utils.go#L236

Reply:
ok， should I close this pr or just left as a issue?

Reply:
This change will break RDMA over ethernet as driver unbind/bind is needed to set the administrative MAC which in turn applies the GUID.

I dont fully understand the issue (why intel nic VF fails to unbind). more information is appreciated. in addition it would be great if you can open an issue for tracking.